docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc

```diff
@@ -179,7 +181,7 @@ The `quarkus.oidc.application-type` property is set to `web-app` in order to tel
 
 Finally, the `quarkus.http.auth.permission.authenticated` permission is set to tell Quarkus about the paths you want to protect.
```

Comment:
```suggestion
Finally, the `quarkus.http.auth.permission.authenticated` permission is set to tell Quarkus about the paths you want to protect.
```

Reply:
@MichalMaler - not sure what was changed here?

Reply:
Not sure how this get into creation of a diff. Feel free to close this one :)

Reply:
I believe this happens when the writer's editor is configured to trim whitespace from the end of the line (the standard practice), and GitHub is configured to hide whitespace changes.

Reply:
Thanks for clarifying @rolfedh and thanks @MichalMaler also for your review and clarification.